src/twig.timezones.js

```diff
@@ -0,0 +1,123 @@
+    IN: ['Asia/New Delhi'],
+    ID: ['Asia/Jakarta'],
+    JO: ['Middle East/Amman'],
+    IQ: ['Middle East/Bagdad'],
+    IR: ['Middle East/Tegeran'],
+    ES: ['Europe/Madrid'],
+    IT: ['Europe/Rome'],
+    KZ: ['Asia/Astana'],
+    KH: ['Asia/Phnom Penh'],
+    CM: ['Africa/Yaounde'],
+    CA: ['North America/Toronto'],
+    QA: ['Middle East/Doha'],
+    KE: ['Africa/Nairobi'],
+    KG: ['Asia/Bishkek'],
+    CN: ['Asia/Beijing'],
+    KP: ['Asia/Pyongyang'],
+    CO: ['South America/Bogota'],
+    CR: ['America/San Jose'],
+    CI: ['Africa/Yamoussoukro'],
+    CU: ['North America/Havana'],
+    KW: ['Middle East/El Kuwait'],
+    LA: ['Asia/Vientiane'],
+    LV: ['Europe/Riga'],
+    LV: ['Africa/Maseru'],
+    LR: ['Africa/Monrovia'],
+    LB: ['Middle East/Beirut'],
+    LY: ['Africa/Tripoli'],
+    LT: ['Europe/Vilnius'],
+    LI: ['Europe/Vaduz'],
+    LU: ['Europe/Luxembourg'],
+    MU: ['Oceania/Port louis'],
+    MR: ['Africa/Nouakchott'],
+    MG: ['Africa/Antananarivo'],
+    MW: ['Africa/Malawi'],
+    MY: ['Asia/Kuala Lumpur'],
+    ML: ['Afrika/Bamako'],
+    MV: ['Asia/Male'],
+    MT: ['Europe/Valetta'],
+    MA: ['Africa/Rabat'],
+    MH: ['Oceania/Majuro'],
+    MX: ['Cental America/Mexico'],
+    MZ: ['Africa/Maputo'],
+    MD: ['Europe/Kishinev'],
+    MN: ['Asia/Ulaanbaatar'],
+    MM: ['Asia/Naypyidaw'],
+    NA: ['Africa/Windhoek'],
+    NR: ['Oceania/Yaren'],
+    NP: ['Asia/Kathmandu'],
+    NE: ['Africa/Niamey'],
+    NG: ['Africa/Abuja'],
+    NL: ['Europe/Amsterdam'],
+    NI: ['Central America/Managua'],
+    NZ: ['Oceania/Wellington'],
+    NO: ['Europe/Oslo'],
+    AE: ['Middle East/Abu Dhabi'],
+    OM: ['Middle East/Muscat'],
+    PK: ['Asia/Islamabad'],
+    PW: ['Oceania/Ngerulmud'],
+    PA: ['Central America/Panama'],
+    PG: ['Oceania/Port Moresby'],
+    PY: ['South America/Asuncion'],
+    PE: ['South America/Lima'],
+    PL: ['Europe/Warsaw'],
+    PY: ['Europe/Lisbon'],
+    CG: ['Africa/Brazzaville'],
+    KR: ['Asia/Seoul'],
+    RU: ['Europe-Asia/Moscow'],
+    RW: ['Africa/Kigali'],
+    RO: ['Europe/Bucharest'],
+    SV: ['Central America/San Salvador'],
+    WS: ['Oceania/Apia'],
+    SM: ['Europe/San Marino'],
+    ST: ['Africa/Sao Tome'],
+    SA: ['Middle East/Riyadh'],
+    MK: ['Europe/Skopje'],
+    SN: ['Africa/Dakar'],
+    RS: ['Europe/Belgrade'],
+    SG: ['Asia/Halima Jacob'],
+    SY: ['Middle East/Damascus'],
+    SK: ['Europe/Bratislava'],
+    SI: ['Europe/Ljubljana'],
+    SO: ['Africa/Mogadishu'],
+    SD: ['Africa/Khartoum'],
+    SR: ['South America/Paramaribo'],
+    US: ['North America/Washington'],
+    TJ: ['Asia/Dushanbe'],
+    TH: ['Asia/Bangkok'],
+    TN: ['Africa/Tunisia'],
+    TM: ['Asia/Ashgabat'],
+    TR: ['Middle East/Ankara'],
+    UZ: ['Asia/Tashkent'],
+    UA: ['Europe/Kyiv'],
+    UY: ['South America/Montevideo'],
+    PH: ['Asia/Manila'],
+    FR: ['Europe/Paris'],
+    HR: ['Europe/Zagreb'],
+    CZ: ['Europe/Prague'],
+    JP: ['Asia/Tokyo']
+}
```

Comment:
Some countries 
have a same zone
have a lot of zones, ex. Russia have 11 zones